src/main/java/com/autotune/utils/CloudWatchLogSender.java

```diff
@@ -0,0 +1,128 @@
+package com.autotune.utils;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.Appender;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.builder.api.*;
+import org.apache.logging.log4j.core.config.builder.impl.BuiltConfiguration;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient;
+import software.amazon.awssdk.services.cloudwatchlogs.model.*;
+
+import java.util.List;
+
+import static com.autotune.operator.KruizeDeploymentInfo.*;
+
+public class CloudWatchLogSender {
+    private static final Logger LOGGER = LogManager.getLogger(CloudWatchLogSender.class);
+
+    private CloudWatchLogSender() {
+    }
+    public static void configureCloudWatchLog() {
+
+        // Check if access details are provided
+        if (cloudwatch_logs_access_key_id != null && !cloudwatch_logs_access_key_id.isEmpty() && cloudwatch_logs_secret_access_key != null && !cloudwatch_logs_secret_access_key.isEmpty() && cloudwatch_logs_region != null && !cloudwatch_logs_region.isEmpty()) {
+            try {
+                // Define default values for attributes if they are empty or null
+                String cw_logs_log_group = cloudwatch_logs_log_group == null || cloudwatch_logs_log_group.isEmpty() ? "kruize-logs" : cloudwatch_logs_log_group;
+                String cw_logs_log_stream = cloudwatch_logs_log_stream == null || cloudwatch_logs_log_stream.isEmpty() ? "kruize-stream" : cloudwatch_logs_log_stream;
+                String cw_logs_log_level = cloudwatch_logs_log_level == null || cloudwatch_logs_log_level.isEmpty() ? "INFO" : cloudwatch_logs_log_level;
+
+                // Create a CloudWatchLogsClient with provided credentials and region
+                CloudWatchLogsClient logsClient = CloudWatchLogsClient.builder()
```

Comment:
should be surrounded by try and catch. even if cloud watch fails application should run. But needs to be reported